index.html

```diff
@@ -3025,7 +3041,7 @@ <h4>Relation to the Verifiable Credentials Data Model</h4>
 use of specific JWT-registered <a>claim</a> names and specific JWS-registered
 header parameter names to allow systems based on JWT to comply with this
 specification. If these specific <a>claim</a> names and header parameters are
-present, their respective counterpart in the standard 
+present, their respective counterpart in the standard
```

Comment:
This looks like a spurious change

Reply:
Some of the folks that provide PRs don't have "strip trailing whitespace" turned on in their text editor. Rather than being super pedantic about it, I tend to just accept those PRs, knowing that my editor will eventually strip that trailing whitespace. It is unfortunate that doing so makes it look like there are spurious changes, but I have been unable to train everyone that produce PRs in proper trailing whitespace hygiene :).